dist/cms.js

```diff
@@ -151,12 +151,57 @@ function getWordPressContentType() {
   return content;
 }
 
+/**
+ * Obtains data about Interactivity API usage on the page.
+ *
+ * @returns {object} Object with fields `total_regions` and `total_regions_by_namespace`.
+ */
+function getInteractivityAPIUsage() {
+  // Look for data-wp-interactive regions.
+  const interactiveRegions = document.querySelectorAll('[data-wp-interactive]');
+
+  // Count the regions by namespace.
+  const regionNamespaceCounts = {}
+  interactiveRegions.forEach( region => {
+    // Extract the namespace from JSON or as a string
+    let namespace = '';
+    const regionAttribute = region.getAttribute('data-wp-interactive');
+    try {
+      const regionData = JSON.parse( regionAttribute );
+      if ( regionData.namespace && 'string' === typeof regionData.namespace ) {
+        namespace = regionData.namespace;
+      }
+    } catch ( e ) {
+      namespace = regionAttribute;
+    }
+    if ( '' !== namespace ) {
```

Comment:
Note that it's also possible for `data-wp-interactive` to contain nothing, in that case it would just be a boolean attribute. So to avoid e.g. something like `true` in the data, we should include something like:
```suggestion
    if ( 'string' === typeof namespace && '' !== namespace ) {
```

Reply:
added

Reply:
I also added some test blocks to the test page and verified the counts still look right:
![image](https://github.com/HTTPArchive/custom-metrics/assets/2676022/b9bb80b5-4619-4a3c-8620-9c5989bad814)